pkg/controller/scyllacluster/resource/resource.go

```diff
@@ -243,26 +273,21 @@ func StatefulSetForRack(r scyllav1.RackSpec, c *scyllav1.ScyllaCluster, sidecarI
 							Command: []string{
 								path.Join(naming.SharedDirName, "scylla-operator"),
 								"sidecar",
+								fmt.Sprintf("--secret-name=%s", naming.AgentAuthTokenSecretName(c.Name)),
+								// TODO: make it configurable
+								"--loglevel=2",
 							},
 							Env: []corev1.EnvVar{
 								{
-									Name: "POD_NAME",
+									Name: naming.OperatorEnvVarPrefix + "SERVICE_NAME",
```

Comment:
nit: sidecar accept 3 mandatory parameters, 2 of them are set here as envvars, thrid once as command line parameter. For consistency i would stick with env vars.

Reply:
ok, i'll unify on flags for consistency. If you mistype a flag name (or it changes) the app will fail, env vars can be silently ignored, so I'd not want to grow optional parameters through env vars.